java/squeek/appleskin/helpers/FoodHelper.java

```diff
@@ -52,7 +52,11 @@ public static BasicFoodValues getDefaultFoodValues(ItemStack itemStack)
 		FoodItemSetting itemFood = itemStack.getItem().getFoodSetting();
 		int hunger = itemFood.getHunger();
 		float saturationModifier = itemFood.getSaturationModifier();
-
+		if (itemStack.getItem() instanceof DynamicFood) {
```

Comment:
This should go in `getModifiedFoodValues` instead. Items can already modify their default values using the standard methods, right?